public/assets/js/publishFunctions.js

```diff
@@ -22,28 +22,28 @@ function validateFile(file) {
 			}
 			break;
 		default:
-			throw new Error('The ' + file.Type + ' content type is not supported.  Only, .jpeg, .png, .gif, and .mp4 files are currently supported.')
+			throw new Error(file.type + ' is not supported a supported file type. Only, .jpeg, .png, .gif, and .mp4 files are currently supported.')
 	}
+	// validate the file name (note: different from the lbry claim name)
```

Comment:
@98farhan94 This function is used to validate the file on the client side before it is submitted.  I added a check to make sure there are no special characters in the file name.  Unfortunately, the file.name in the file object is read only so we can't simply replace the spaces.  For now we will just prompt the user to fix their file name.